scripts/tools/zap_regen_all.py

```diff
@@ -420,6 +420,11 @@ def getGlobalTemplatesTargets():
 def getCodegenTemplates():
     targets = []
 
+    targets.append(JinjaCodegenTarget(
```

Comment:
was there a reason for this move? Otherwise not moving would result in smaller code delta..